hydro_deploy/hydro_cli_examples/hydro_python_sender_azure.hydro.py

```diff
@@ -0,0 +1,60 @@
+import hydro
+from pathlib import Path
+import time
+
+async def main(args):
+    machine_azure = args[0] == "azure"
+
+    deployment = hydro.Deployment()
+    localhost_machine = deployment.Localhost()
+
+    # gcp_vpc = hydro.GCPNetwork(
+    #     project="autocompartmentalization",
+    # )
+
+    machine2 = deployment.AzureHost(
+        project="hydro-example" + str(int(time.time())),
+        os_type="linux",
+        machine_size="Standard_B1s",
+        region="East US",
+        image="",
```

Comment:
If the image can indeed be omitted, we should make this an optional parameter.

Reply:
The azure image parameter is a little lengthy, so I made it optional for the user.

https://arc.net/l/quote/moucnywd